src/main/kotlin/org/rust/lang/core/psi/ext/RsTraitItem.kt

```diff
@@ -76,6 +76,20 @@ private val RsTraitItem.superTraits: Sequence<BoundElement<RsTraitItem>> get() {
     return bounds.mapNotNull { it.bound.traitRef?.resolveToBoundTrait }
 }
 
+val RsTraitItem.isSized: Boolean get() {
+    val stub = stub
+    if (stub != null) return implementedTrait?.flattenHierarchy.orEmpty().any { it.element.isSizedTrait }
```

Comment:
Hmm. Obviously your solution will work only if the file with the trait is opened in an editor. Really you will not need this special case if write the code without accessing non-stubbed elements and a text of elements. (you can read more about stubs in ARCHITECTURE.md).